rerun

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
If this is part of the test infrastructure, I feel that this should be moved to `tests/contrib` or `tests/bin`. Is there a compelling reason to keep it at the project root?